### ISSUE TITLE
Initial sync fails if server doesn't respond within 30 seconds

### ISSUE THREAD

**Issue by garethbowen** [medic/medic-webapp#2771]
With some users initial replication isn't responding quickly enough and the request times out. This means the app loads with lots of errors. Eventually the background replication should fix the app and a refresh will get back to normal but it's a jarring experience. It should have a heartbeat so timeout doesn't occur unless the server is unreachable.

Affects 2.8.0

Causes #2770

**Comment by garethbowen** [comment]
Done. I picked a timeout of 10 minutes trying to balance between a number that's too small resulting in an ugly first run experience, and a number that's too large forcing users to watch the spinner for an hour before the app loads at all.

The previous default was 30 seconds which was causing an issue for a specific user who, after curling the changes feed, I determined was taking ~ 1 minute to get a response. So 10 minutes will safely accommodate that user.
